src/components/tab/index.tsx

```diff
@@ -0,0 +1,43 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import './index.scss';
+
+interface TabProps {
+    children?: React.ReactNode;
+    onClick?: (tabIndex: number) => void;
+    tabIndex: number;
+    className?: string;
+    active?: boolean;
+    disabled?: boolean;
+    linkClassName?: string;
+}
+
+export const Tab: React.FC<TabProps> = ({
+    children,
+    className,
+    onClick,
+    tabIndex,
+    active,
+    disabled,
+    linkClassName,
+}) => {
+    const cnLink = classNames(`tab-link ${linkClassName}`, {
+        'active': active,
```

Comment:
use BEM in all styles. Check out example in https://github.com/openware/react-components/tree/master/src/components/landing-block